testsuite/systemtest/bintools/BlocksTools/blocks_translator.testdesc

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Can you use the new test format tcetest_*.sh instead?

Reply:
Updated!